src/js/popup.js

```diff
@@ -319,6 +319,13 @@ $(document).ready(function () {
         loadTemplateEditor();
     });
 
+    if ($('.navbar-fixed').width() < $('.custom-domain-list').width() ) { // eslint-disable-line space-in-parens, key-spacing
+        $('.navbar-fixed').css({'width':($('.custom-domain-list').width() + 'px')}); // eslint-disable-line space-in-parens, key-spacing
+    } else {
+        $('.custom-domain-list').css({'width':($('.navbar-fixed').width() + 'px')}); // eslint-disable-line space-in-parens, key-spacing
+    }
+    $('.custom-domain-list').css({'max-width':($('.navbar-fixed').width() + 'px')}); // eslint-disable-line space-in-parens, key-spacing
```

Comment:
This all seems very unnecessary. I'm pretty sure we could get around all of this by simply adding max-width: 400px to https://github.com/rdbrck/jira-template-injector/blob/master/src/css/popup.css#L7

Or potentially even easier changing `min-width` -> `width`

Reply:
:+1: